server/src/main/kotlin/suwayomi/tachidesk/manga/impl/backup/proto/models/BackupServerSettings.kt

```diff
@@ -48,6 +48,7 @@ data class BackupServerSettings(
     @ProtoNumber(29) override var basicAuthEnabled: Boolean,
     @ProtoNumber(30) override var basicAuthUsername: String,
     @ProtoNumber(31) override var basicAuthPassword: String,
+    @ProtoNumber(56) override var cookieAuthEnabled: Boolean,
```

Comment:
Please also check that this is fine. I'm assuming that the number here is important for deserialization of backups, and that renumbering would cause old backups to fail to deserialize properly, is that right?

Reply:
Correct, whenever a new setting is added, it needs to use the next number available.